navigation/dead_reckoning/sam_dead_reckoning/launch/sam_dr_launch.py

```diff
@@ -4,88 +4,71 @@
 from ament_index_python.packages import get_package_share_directory
 from launch_ros.actions import Node
 
+# TODO xml version of launch file
 
 def generate_launch_description():
-    # Example: using a yaml file for parameters
-    # param_config = os.path.join(
-    #     get_package_share_directory("sam_graph_slam_2"),
-    #     "config",
-    #     "pipeline_config.yaml"
-    # )
-
-    # TODO this is not the correct way to set the name
-    # I think we should talk about how launch files should be set up
-    robot_name = "sam0"
-    odom_topic = f"/{robot_name}/dr/odom"
-    gps_odom_topic = f"/{robot_name}/dr/gps_odom"
+    namespace = "sam0"
+    # These topics are specific to sam dr
+    depth_topic = f"{namespace}/dr/depth"
+    odom_topic = f"/{namespace}/dr/odom"
+    gps_odom_topic = f"/{namespace}/dr/gps_odom"
 
     print("Launching sam_dr_launch.py")
 
     return launch.LaunchDescription([
-        # Example: ExecuteProcess
-        # ExecuteProcess(
-        #     cmd=["ros2", "bag", "play", "/home/julian/bag_files/pipeline_sim"],
-        #     output="screen"
-        # ),
-        # TODO include the latlon to utm service
+        # TODO include the latlon to utm service ??
         Node(
             package="sam_dead_reckoning",
             executable="depth_node",
+            namespace=namespace,
             name="depth_node",
             output="screen",
             parameters=[{
-                "odom_frame":  f"{robot_name}/odom",
-                "base_frame": f"{robot_name}/base_link",
-                "pressure_frame": f"{robot_name}/pressure_link",
-                "pressure_topic": f"/{robot_name}/core/depth20_pressure",
-                "depth_topic": f"/{robot_name}/dr/depth",
+                "odom_frame":  f"{namespace}/odom",
```

Comment:
The sim uses `_`, and it can not use `/` due to unity related reasons. Which means we'll need to use `_` for the real sam as well. So these should probably be `f"{namespace}_odom`.

I saw that you HAVE changed some of the frame names below. Maybe do a search-and-replace over the repos to make sure you havent missed any?

In fact, since these link names are also shared like topic names, they should live in a message like the topics. 
`messages/dr_msgs/msg/link_names.msg` is probably a good place for them?

Also can you keep a list of all the frames used as you use them in the readme? Not all of them are there in the sim or they are there differently and that should change to match the DR stack to the sim.